add note for Azure Availbility Zone (#66200)
ansible/ansible: docs/docsite/rst/scenario_guides/guide_azure.rst

        version: latest

> Distribute Virtual Machines in Availability Zones

Reviewer: I prefer the title like `Creating a Virtual Machine in Availability Zones`, two reasons:
1. This can be consistent with above titles like `Creating a Virtual Machine with Default Options`.
2. Tutorial like [this](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/create-portal-availability-zone) has similar title.

If this suggestion is taken, please also update related words in the content, thanks.

Author: Thank you for the review. I will update.

..................................................

> If you want to create a Virtual Machines in Availability Zones,

Reviewer 2: ```suggestion
If you want to create a VM in an availability zone,
```

> Note the following:

Reviewer 2: ```suggestion
consider the following:
```

> * Both OS disk and data disk must be a 'Managed Disk', not 'Unmanaged Disk',

Reviewer 2: ```suggestion
* Both OS disk and data disk must be a 'managed disk', not an 'unmanaged disk'.
```

>   to create a Virtual Machine in Availability Zones.

Reviewer 2: ```suggestion
```

> * When creating a Virtual Machine with the azure_rm_virtualmachine,

Reviewer 2: ```suggestion
* When creating a VM with the ``azure_rm_virtualmachine`` module,
```

>   you need to specify explicitly managed_disk_type

Reviewer 2: ```suggestion
  you need to explicitly set the ``managed_disk_type`` parameter
```

>   to change OS disk to 'Managed Disk'.

Reviewer 2: ```suggestion
  to change the OS disk to a managed disk.
```

>   Otherwise, OS disk becomes 'Unmanaged disk'.

Reviewer 2: ```suggestion
  Otherwise, the OS disk becomes  an unmanaged disk..
```

> * When creating a data disk with azure_rm_manageddisk,

Reviewer 2: ```suggestion
* When creating a data disk with azure_rm_manageddisk,
```
```suggestion
* When you create a data disk with  the ``azure_rm_manageddisk`` module,
```

>   you need to specify explicitly storage_account_type

Reviewer 2: ```suggestion
  you need to  explicitly specify the  ``storage_account_type`` parameter
```

>   to make it a 'Managed Disk'.

Reviewer 2: ```suggestion
  to make it a  managed disk.
```

>   Otherwise, the data disk will be 'Unmanaged disk'.

Reviewer 2: ```suggestion
  Otherwise, the data disk will be an unmanaged disk.
```

> * 'Managed Disk' does not require 'Storage Account' or 'Storage Container'

Reviewer 2: ```suggestion
* A managed disk does not require a storage account or a storage container,
```

>   unlike 'Unmanaged Disk'.

Reviewer 2: ```suggestion
  unlike  a n unmanaged disk.
```

>   In particular, note that once a Virtual Machine is created on an 'Unmanaged disk',

Reviewer 2: ```suggestion
  In particular, note that once a VM is created on an unmanaged disk,
```

>   an unnecessary 'Storage container' named "vhds" is automatically created.

Reviewer 2: ```suggestion
  an unnecessary storage container named "vhds" is automatically created.
```

> * When creating an IP address with azure_rm_publicipaddress,

Reviewer 2: ```suggestion
* When you create an IP address with the ``azure_rm_publicipaddress`` module,
```

>   it is necessary to specify 'standard' in the 'sku' property.

Reviewer 2: ```suggestion
  you must set the  ``sku`` parameter to ``standard``.
```

>   Otherwise, the IP address cannot be used in Availability Zones.

Reviewer 2: ```suggestion
  Otherwise, the IP address cannot be used in an availability zone.
```
